### DISCUSSION TITLE
Downloading PDF (or any file) not working

### DISCUSSION THREAD

**Discussion by emmenru** [knowledgefutures/pubpub#1263] [Support] [answered]
Hello, 

I am experiencing issues with downloading a PDF (or actually being able to download any file) from my publication. It worked yesterday, but now I seem to only get the message "there was a problem generating your file" and no file is downloading. Any ideas on how this can be fixed, or if there is any other way where I can access my publication? 

Thanks!

**Comment by gabestein** [comment, marked as answer]
My sincere apologies for the issues today. We're facing what is either a runaway process from a few greedy browsers or a more malicious DDoS-style attack, centered on NIME. It's causing all of our background processes, primarily import and export, to fail.
​
We've been working on it since last night, when we were able to briefly stem the tide, and are continuing to make progress this morning. I'll let you know as soon as we know more.

**Reply by kitchWWW** [reply]
Thank you for the context on this! was running in the same problem just now.

**Reply by emmenru** [reply]
Thank you for your kind reply and this information!

**Reply by gabestein** [reply]
Okay folks, we've pushed a fix for this issue. You should try refreshing any open pages, and then exporting again. If the issue persists, try making a single change to your Pub (a space, a line break, etc.) and then exporting again. We're still clearing our worker queue cache, which is causing some lingering issues.

Again, my apologies for the issues.

**Reply by emmenru** [reply]
Thanks, now it works!
